keras_nlp/models/preprocessor.py

```diff
@@ -96,6 +97,13 @@ def from_preset(
         )
         return cls(tokenizer=tokenizer, **kwargs)
 
+    def save_to_preset(
```

Comment:
I actually thing we have a bug here in the format of our one toy classification preset.

https://www.kaggle.com/models/keras/bert/frameworks/keras/variations/bert_tiny_en_uncased_sst2

Basically `BertClassifier.from_preset("bert_tiny_en_uncased_sst2").preprocessor` is not always the same as `BertPreprocessor.from_preset("bert_tiny_en_uncased_sst2")`. Or rather, it is currently, but only because the preprocessing layer happens to have all default parameters. If a preprocessor had custom config options when we saved that would not currently get reflected during preprocessor load.

We need to clean this up as part of our saving flow whatever we come up with. Save a `preprocessor.json`? Not sure, but let's discuss.

Reply:
Thanks for bringing this up! I think adding a `preprocessor.json` is a good idea because it seems that we need it to be able to support saving/loading preprocessor separately.